rosdep/base.yaml

```
@@ -8066,6 +8066,11 @@ xtensor:
   ubuntu:
     bionic: [xtensor-dev]
     jammy: [libxtensor-dev]
+xsimd:
+  debian: [libxsimd-dev]
```

Since this is only available in upcoming versions of Debian, we should mark this as unavailable in currently supported versions:

```suggestion
  debian:
    "*": [libxsimd-dev]
    bullseye: null
    buster: null
```